src/createEpicMiddleware.js

```diff
@@ -45,6 +48,7 @@ export function createEpicMiddleware(epic, options = defaultOptions) {
           return output$;
         })
         ::switchMap(output$ => options.adapter.output(output$))
+        ::mergeMap(action => of(action, asap))
```

Comment:
oh right cause observeOn will schedule _everything_ whereas mergeMap+of(a, asap) will just schedule next, not error/complete